### ISSUE TITLE
How to always display CursorLine and CursorCrosshair

### ISSUE THREAD

**Issue by dima-devs** [coinjar/react-native-wagmi-charts#89]
## Description
It's possible to have an option to active `CursorLine` and `CursorCrosshair` on current `Index`? 
Current these `Cursor`s are showing when click or move on Chart.

Is there any option to enable this? 
Who can help with this problem? Thank you in advance.

## Version
`"react-native-wagmi-charts": "2.1.0"`

**Comment by felix-lambert** [comment]
@dima-devs did you manage to solve this problem ?

**Comment by dima-devs** [comment]
No, I couldn't solve the problem yet. Do you have any update? @felix-lambert

**Comment by felix-lambert** [comment]
Hello I did a fork of this repo > @epsor/react-native-wagmi-charts. Please do 

`yarn add @epsor/react-native-wagmi-charts@2.1.1`

Then you can try:

```
<LineChart.CursorLine
    holdValue={true}
/>
```

Please tell me if it works

**Comment by dima-devs** [comment]
@felix-lambert Great Job! It works for me as well.
Can I set initial value to hold after load the chart? this will show the cursor line after load data without touch the chart.
Now, I can only see the cursor line by touch once. make sense what I mean?

**Comment by felix-lambert** [comment]
Yes you can update the new repo to the last version and do >

```
<LineChart.CursorLine
        holdValue={true}
        mountWithActivatedCursor={true}
      >
```

Please tell me if it works.

**Comment by dima-devs** [comment]
@felix-lambert  Great! it works, but `inactiveColor` props is not working.

**Comment by felix-lambert** [comment]
Can you try this version ? 

`yarn add @epsor/react-native-wagmi-charts@2.1.4`

Best

**Comment by dima-devs** [comment]
@felix-lambert yes, `v2.1.6` is not working, 
But `v2.1.4` is works,

**Comment by Working4u** [comment]
How can we import @epsor/react-native-wagmi-charts and use it?

**Comment by barrymichaeldoyle** [comment]
Could you make a PR into this main repo?

**Comment by darkhorse-coder** [comment]
Sure @barrymichaeldoyle if you still need this change, I can create PR.

**Comment by darkhorse-coder** [comment]
> Could you make a PR into this main repo?

Hi @barrymichaeldoyle why I don't have permission to create a branch and push??

**Comment by honeybadger26** [comment]
Hi @darkhorse-coder. Thanks for offering to make a PR! You can create a fork of this repo and submit a PR from the forked repo. More info [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
